src/styles/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: $star-size;
+    width: $star-size;
+    background-image: url($star-default);
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__star:not(:last-of-type) {
+    margin-right: $star-spacing;
+  }
+
+  &--0 .stars__star.stars__star:nth-child(-n + 0),
+  &--1 .stars__star.stars__star:nth-child(-n + 1),
+  &--2 .stars__star.stars__star:nth-child(-n + 2),
+  &--3 .stars__star.stars__star:nth-child(-n + 3),
+  &--4 .stars__star.stars__star:nth-child(-n + 4),
+  &--5 .stars__star.stars__star:nth-child(-n + 5) {
+    background-image: url($star-active);
```

Comment:
The variables `$star-size`, `$star-spacing`, `$star-default`, and `$star-active` are used but not defined in this file. Ensure these variables are defined in a variables file or imported correctly to avoid compilation errors.